lib/capybara/rspec/matchers.rb

```diff
@@ -268,3 +268,27 @@ def become_closed(options = {})
     end
   end
 end
+
+if defined?(::RSpec::Matchers)
```

Comment:
Currently this code is executed only if user requires `capybara/rspec/matchers`.
But the user may just include `Capybara::DSL` and `RSpec::Matchers` into the same scope without requiring this capybara file.